Jenkinsfile

```diff
@@ -303,393 +307,61 @@ pipeline {
       }
     }
 
-    stage('Update version') {
+    stage('Deploy Preview') {
       steps {
+        setGitHubBuildStatus('nuxeo/preview/deploy', 'Deploy nuxeo Preview', 'PENDING')
```

Comment:
Maybe just `nuxeo/preview`? Shorter in the status list.
I know it's like this in ARender, but WDYT @troger @kevinleturc ?

Reply:
Agree 👍

Reply:
Yes sure 👍